app/main.py

```diff
@@ -1 +1,49 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str,):
```

Comment:
You have a trailing comma here, remove it.

Reply:
Yes, sir!